assets/js/common.js

```diff
@@ -1,4 +1,5 @@
-/* global document */
+/* global document, wp */
+const { __ } = wp.i18n;
```

Comment:
Let's not use ES6 just yet. You could write this also as `var __ = wp.i18n.__`.

But we shouldn't make all the API methods available in the global window object. So do the extra variable only when necessary in the relevant method.

Reply:
Reverted back to ES5